src/main/java/fi/fmi/avi/model/swx/SpaceWeatherAdvisoryAnalysis.java

```diff
@@ -1,13 +1,15 @@
-package fi.fmi.avi.model.SpaceWeatherAdvisory;
+package fi.fmi.avi.model.swx;
 
 import java.util.List;
 import java.util.Optional;
 
+import fi.fmi.avi.model.AviationWeatherMessageOrCollection;
 import fi.fmi.avi.model.PartialOrCompleteTimeInstant;
 
-public interface SpaceWeatherAdvisoryAnalysis {
+public interface SpaceWeatherAdvisoryAnalysis extends AviationWeatherMessageOrCollection {
     PartialOrCompleteTimeInstant getTime();
 
+    //TODO/FIXME: are there any cases where we would not have the type?
     Optional<Type> getAnalysisType();
 
     Optional<List<SpaceWeatherRegion>> getRegion();
```

Comment:
Perhaps we should use `List<SpaceWeatherRegion>` without `Optional` here. I'm not aware that an empty list would be semantically different from a non-existent list. E.g. IWXXM 3.0 schema requires at least one `region` (which may be `nil`).

In addition, maybe we should consider a common `enum NilReason`, containing all NIL reasons specified in WMO [Register: Code Table D-1: Nil reasons](https://codes.wmo.int/common/nil). And then, replacing mutually exclusive `boolean isNoPhenomenaExpected()` and `boolean isNoInformationAvailable()` with a `NilReason` field?